gtk/src/ghb3.ui

```diff
@@ -9306,7 +9306,7 @@ Uncheck this if you want to allow changing each title's settings independently.<
     </child>
     <action-widgets>
       <action-widget response="cancel">source_cancel</action-widget>
-      <action-widget response="ok">source_ok</action-widget>
+      <action-widget response="no">source_ok</action-widget>
```

Comment:
Should this be `response="yes"`, instead?  an Open button with the ID `source_ok`  that sends a `no` response is hurting my brain. (Also, maybe the ID should be `source_open`?)

Reply:
Or, you could use `response="accept"` or `response="reject"`, which according to the docs are both a:
> Generic response id, not used by GTK+ dialogs

But either way I'd definitely change the widget `id`.

Reply:
This is what I thought originally, which is why I changed it [previously](https://github.com/HandBrake/HandBrake/commit/802caf8a78dfd31ee2695bb1b397067bcc17277b). But then I found that it was done deliberately, in order to let you select a folder to scan instead of just a single file.

> For most dialog boxes you can use your own custom response codes rather than the ones in GtkResponseType, but GtkFileChooserDialog assumes that its “accept”-type action, e.g. an “Open” or “Save” button, will have one of the following response codes:
> 
> GTK_RESPONSE_ACCEPT
> 
> GTK_RESPONSE_OK
> 
> GTK_RESPONSE_YES
> 
> GTK_RESPONSE_APPLY
> 
> This is because GtkFileChooserDialog must intercept responses and switch to folders if appropriate, rather than letting the dialog terminate — the implementation uses these known response codes to know which responses can be blocked if appropriate.

Basically if you select a folder and press Open, the dialog closes, thinking it was cancelled, but HandBrake still gets the selected folder and tries to open it. It may not be very intuitive, but it saves having a separate menu item for opening a folder. Having said that, using a non-standard code (but not `accept`) also seems to work, so I could change it to that instead. 

Also, in my other pull request which switches the file dialogs to GtkFileChooserNative, I had to add an Open Directory menu item anyway as it doesn't let you set the response codes. So this could end up being irrelevant soon.

Reply:
Ah! That makes sense. (Insofar as anything in Gtk makes sense, I mean. It's all relative.)

No harm in using "no" then, IMHO, though It'd definitely be less brain-straining if the widget ID was changed from `source_ok` to `source_open`. (Maybe even with a `<!-- … -->` comment in the `.ui` file, explaining that `source_open` needs to send a `no` signal to fake folder-selection semantics.)